Documentation/ReleaseNotes.md

```diff
@@ -185,8 +185,19 @@ Motion Controller simulation is now offered in editor play mode along side the e
 
 ### Known issues
 
+**Oculus Quest**
+
 There is currently a known issue for using the [Oculus XR plugin with when targetting Standalone platforms](https://forum.unity.com/threads/unable-to-start-oculus-xr-plugin.913883/).  Check the Oculus bug tracker/forums/release notes for updates.
 
 The bug is signified with this set of 3 errors:
 
 ![Oculus XR Plugin Error](https://forum.unity.com/attachments/erori-unity-png.644204/)
+
+**UnityUI and TextMeshPro**
+
+There's a known issue for newer version of TextMeshPro (2.1.1+) and Unity 2019.4.x, where the default font size for dropdowns has been increased
```

Comment:
Issue also happens with 1.5.0+. Its independent of Unity version. 
Apart from dropdown font size, the character spacing on second column is also altered as you can see in the image.